Modules/Numerics/Statistics/test/itkHistogramTest.cxx

```diff
@@ -97,51 +96,117 @@ itkHistogramTest(int, char *[])
 
   // Query the bounds of the bin using the index of the bin.
 
-  if (itk::Math::NotAlmostEquals((lowerBound[0] + interval * 31), histogram->GetBinMin(0, 31)))
+  const MeasurementType & expectedValMeasType1 = lowerBound[0] + interval * 31;
+  const MeasurementType & obtainedValMeasType1 = histogram->GetBinMin(0, 31);
+  if (itk::Math::NotAlmostEquals(obtainedValMeasType1, expectedValMeasType1))
```

Comment:
If we can't use GTests's `ASSERT_NEAR` or something else, we might want to create our own testing macro to avoid boilerplate repetition.

Reply:
This was already mentioned in issue #1273. I am not in favor of using 2 testing systems, but that seems a lost battle. GTests are still not even mentioned in the ITK SWG if I'm not mistaken.

Reply:
Reviewing these repetitive checks is not easy, and writing them is even harder. If you are investing effort into these cleanups and improvements, creating a macro seems like an investment which will "pay itself off" withing a few PRs 😄

Reply:
I am aware of this Dženan, especially because I go over these pieces. I would like to revisit the issue as time permits (long-term). Resolving now.